internal/eva/organization_unit.go

```diff
@@ -123,18 +123,18 @@ type GetOrganizationUnitDetailedRequest struct {
 }
 
 type GetOrganizationUnitDetailedResponse struct {
-	ID                  int64   `json:"ID"`
-	Name                string  `json:"Name"`
-	PhoneNumber         string  `json:"PhoneNumber"`
-	EmailAddress        string  `json:"EmailAddress"`
-	ParentID            int64   `json:"ParentID"`
-	CurrencyID          string  `json:"CurrencyID"`
-	BackendID           string  `json:"BackendID"`
-	CostPriceCurrencyID string  `json:"CostPriceCurrencyID"`
-	Type                int64   `json:"Type"`
-	Latitude            float64 `json:"Latitude"`
-	Longitude           float64 `json:"Longitude"`
-	Address             Address `json:"Address"`
+	ID                  int64    `json:"ID"`
```

Comment:
I think everything is optional except `ID` and `ParentId` 🤔 
I think Go sets default values for empty primitives, so string would be empty string and numbers would be zero value. Do we want to handle these case?

Reply:
Yes, we will need to handle these case differently, as we spoke yesterday we will need to use `pointers` for every attribute that can be nullable/omitted. I will tackle this in another PR.